src/server/troubleshoot/index.md

```diff
@@ -0,0 +1,19 @@
+# Troubleshoot
```

Comment:
I would consider naming this something like "Troubleshoot custom servers" as there is already a page called [Troubleshoot](https://merginmaps.com/docs/misc/troubleshoot/). It would be easier to find what users are looking for when using the search bar.